CONTRIBUTING.md

```diff
@@ -1,59 +1,86 @@
-# Contributing to CircuitVerse
-[Back to `README.md`](README.md)
+# Contributing to Circuitverse
 
-Thanks for checking out our contribution guide. We greatly appreciate any work contributed, no matter how small!
+- We expect contributors to abide by our underlying [Code of Conduct](https://github.com/CircuitVerse/CircuitVerse/blob/master/code-of-conduct.md) . All discussions about this project must be respectful and harassment-free.
+- Remember that communication is the lifeblood of any Open Source project. We are all working on this together, and we are all benefiting from this software.
+- It's very easy to misunderstand one another in asynchronous, text-based conversations. When in doubt, assume everyone has the best intentions.
+- If you feel anyone has violated our Code of Conduct, you should anonymously contact the team with our abuse report form via [Slack](https://circuitverse.org/slack), necessary action will be take by the team.
 
-## How to Contribute
-When you are ready to start work on an issue:
+## Where to contribute
 
-- Let us know by leaving a comment on the issue. (Also let us know later if you are no longer working on it.)
-- Once you are assigned the issue (or once you have claimed the issue) only then proceed to make the Pull Request. This will help avoid multiple PRs pertaining to the same issue.
+Please note if someone is already assigned or working on the issue, do not try to start working without asking in a thread.
 
-If you don't see your idea listed, do one of the following:
-* **If your contribution is minor,** such as a typo fix, open a pull request.
-* **If your contribution is major,** such as a new feature/enhancement, start by opening an issue first. This way, other people can be also involved in the discussion before you do any work.
+- `core-team` label are internal tasks that will be completed by a Circuitverse core team member. 
```

Comment:
After a core team member updates the labels please insert the label link here also.

Reply:
Sure

Reply:
What about using the term "maintainers" instead of core team in this document ?